src/repository/screens/commit-list.screen.js

```diff
@@ -24,29 +24,30 @@ class CommitList extends Component {
     return item.id;
   };
 
-  renderItem = ({ item }) =>
-    <CommitListItem commit={item} navigation={this.props.navigation} />;
+  renderItem = ({ item }) => (
+    <CommitListItem commit={item} navigation={this.props.navigation} />
+  );
```

Comment:
Can we do some prettier/eslint configurations so that these changes will not be made? As well as the space after `async` and extra empty line between every let/const.